pizza_recipe.txt

```diff
@@ -8,7 +8,6 @@ Mix liquid in a 2.5-quart bowl:
 	6� cups water
 	�  cup Oil
 
-Dump 5-pound bag of flour on counter.
```

Comment:
I agree this is horrible.  However, we still need flour.  I would suggest adding 
```
2 cups of flour
```